### ISSUE TITLE
Mod won't launch

### ISSUE THREAD

**Issue by d3x47l0n** [Mantas22XYT/RDSquared#1]
I don't know why
```
(00:27:24) Java version: Oracle Corporation, Java(TM) SE Runtime Environment, 1.8.0_371-b11
(00:27:24) System: windows 7, 6.1, x86
(00:27:24) Version RubyDungSquared has an invalid `file-ver` parameter.
(00:27:24) Loaded in: 175540194 ns
(00:27:24) java.io.FileNotFoundException: C:\Users\ASUS\Desktop\.betacraft\launcher\launch-methods\preclassic
(00:27:24) .jar (The filename, directory name, or volume label syntax is incorrect)
(00:27:24) 	at java.util.zip.ZipFile.open(Native Method)
(00:27:24) 	at java.util.zip.ZipFile.<init>(Unknown Source)
(00:27:24) 	at java.util.zip.ZipFile.<init>(Unknown Source)
(00:27:24) 	at java.util.jar.JarFile.<init>(Unknown Source)
(00:27:24) 	at java.util.jar.JarFile.<init>(Unknown Source)
(00:27:24) 	at org.betacraft.launcher.Launcher.loadClasses(Launcher.java:296)
(00:27:24) 	at org.betacraft.launcher.Launcher.main(Launcher.java:219)
(00:27:24) Launch method: preclassic
(00:27:24) 
(00:27:24) java.lang.ClassNotFoundException: preclassic
(00:27:24) 
(00:27:24) 	at java.net.URLClassLoader.findClass(Unknown Source)
(00:27:24) 	at java.lang.ClassLoader.loadClass(Unknown Source)
(00:27:24) 	at java.lang.ClassLoader.loadClass(Unknown Source)
(00:27:24) 	at org.betacraft.launcher.Launcher.main(Launcher.java:221)
(00:27:25) 
Client closed.

```

**Comment by OksmaWarez** [comment]
probably because i fucked something up when exporting the latest version, i'll check whenever i have time. thanks for reporting

**Comment by OksmaWarez** [comment]
should be fixed in 1.2_01
